### DISCUSSION TITLE
Testing kube-bench on minikube/macos - only logs summary

### DISCUSSION THREAD

**Discussion by maoo** [aquasecurity/kube-bench#1493] [Questions and Help]
I'm trying to test kube-bench on minikube (`v1.31.2`), before running it on prod environments; as I'm running on MacOS, I understand this is the only (documented) way I can run kube-bench.

When minikube is up and running, I install kube-bench using `kubectl apply -f https://raw.githubusercontent.com/aquasecurity/kube-bench/main/job-master.yaml` and I validate the installation using `kubectl get pods -l job-name=kube-bench-master` and `kubectl describe pod -l job-name=kube-bench-master`; so far so good.

But when I access logs - with `kubectl logs -l job-name=kube-bench-master` - I can't see log details, all I can see is this:
```
0 checks FAIL
30 checks WARN
0 checks INFO

== Summary total ==
59 checks PASS
13 checks FAIL
52 checks WARN
0 checks INFO
```

I also tried to play with the container's `command` and set it as `["kube-bench", "run", "--benchmark", "cis-1.24", "-v", "5", "--include-test-output"]` , but got the same result.

I feel like I'm missing something basic here; any help would be greatly appreciated.

Thanks in advance.

**Comment by bindurthi-git** [comment]
Try removing the parameter '-l' and use `kubectl logs kube-bench-master`
